kettle/Dockerfile

```diff
@@ -28,6 +28,7 @@ RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime \
     python \
     python3 \
     python3-pip \
+    vim \
```

Comment:
we usually avoid adding text editors to prod images, i'm guessing this was needed during debugging?

Reply:
Yes it is just generally helpful but I can remove